### ISSUE TITLE
Support Python 3.7

### ISSUE THREAD

**Issue by Crocmagnon** [benleb/ad-automoli#43]
Python 3.7.3 is the current latest version available on Raspbian, on which many people have Home Assistant and AppDaemon installed.

Python 3.8 is not available unless one builds it from source.

Would it be possible to keep supporting Python 3.7 until 3.8 is available in raspbian repos? I found at least one breaking change [in the last release](https://github.com/benleb/ad-automoli/compare/v0.7.2...v0.7.3#diff-bab9593e641fec176fada8f8cbe40415R112)

**Comment by benleb** [comment]
I choose the [AppDaemon Docker Image](https://hub.docker.com/r/acockburn/appdaemon/) as a "reference" which I want to support, sorry.
But compiling Python is no problem. Google for it, it's just a couple of lines to copy & paste and a chance to learn something new 😄 👍

**Comment by Crocmagnon** [comment]
That's sad because as stated in [appdaemon docs](https://appdaemon.readthedocs.io/en/latest/INSTALL.html#raspberry-pi-docker), this docker image isn't directly compatible with the Raspberry Pi. And last time I checked, Docker was very clumsy to install on an existing raspberry pi (if you don't want to format your system and install HypriotOS).

I've already compiled Python many times but:

1. It's slow on a raspberry pi
2. It's not an installation method that's tested by raspbian maintainers
3. It's something else to maintain

Also, supporting Python 3.7 gives you support for Python 3.8, so you'd still support the docker image 😇
